.github/workflows/tests.yml

```diff
@@ -70,3 +70,22 @@ jobs:
       run: |
         go fmt ./...
         git diff --exit-code
+
+  docs:
+    runs-on: ubuntu-latest
+
+    name: Automated docs check
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.10' 
+
+    - name: Ensure no CLI reference doc changes
+      run: |
+        python3 generate_command_docs.py
```

Comment:
I guess we should probably make this `tox -e commands` (and install tox like we do in Ops framework_tests) in case the script starts to grow dependencies.

Reply:
Upgraded to python 3.12 and the latest version of tox (4.17). Triggering CI test now.